_database-integrations/mssql/google-cloud/v1/google-cloud-v1.md

```diff
@@ -0,0 +1,174 @@
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.driver | flatify | strip }}. [TODO]
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true
+
+frequency: "30 minutes"
+tier: "Standard"
+port: 1433
+db-type: "mssql"
+
+## Stitch features
+
+api-type: "platform.mssql"
+versions: "2012 through 2017"
+ssh: true
+ssl: true
+
+## General replication features
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+select-all: false
+select-all-reason: |
+  {{ integration.display_name }} integrations don't currently support a default Replication Method, which is required to use the Select All feature. The default Replication Method setting is only available for integrations that support Log-based Incremental Replication.
+
+table-level-reset: true
+
+## Replication methods
+
+define-replication-methods: true
+
+log-based-replication-minimum-version: "n/a"
+log-based-replication-master-instance: false
+log-based-replication-master-instance-reason: "Google CloudSQL doesn't currently support logical replication."
+log-based-replication-master-instance-doc-link: "https://groups.google.com/forum/#!topic/google-cloud-sql-discuss/md_7Rq3LgB0"
+log-based-replication-read-replica: false
+
+## Other Replication Methods
+
+key-based-incremental-replication: true
+full-table-replication: true
+
+view-replication: true
+
+
+# -------------------------- #
+#      Setup Requirements    #
+# -------------------------- #
+
+requirements-list:
+  - item: "**Permissions in Google Cloud that allow you to modify the database's connection settings.** This is required to whitelist Stitch's IP addresses."
+  
+  - item: "**A database running {{ integration.display_name }} version {{ page.versions }}.** {{ integration.display_name }} 2012 is the miminum version that Stitch supports for this type of integration."
+
+
+# -------------------------- #
+#     Setup Instructions     #
+# -------------------------- #
+
+setup-steps:
+  - title: "Verify your Stitch account's data pipeline region"
+    anchor: "verify-stitch-account-region"
+    content: |
+      {% include shared/whitelisting-ips/locate-region-ip-addresses.html first-step=true %}
+
+  - title: "Configure database connection settings"
+    anchor: "connect-settings"
+    content: |
+      {% include integrations/templates/configure-connection-settings.html %}
+
+  - title: "Create a Stitch database user"
+    anchor: "create-a-database-user"
+    content: |
+      In this step, you'll create a dedicated database user for Stitch. This will ensure Stitch is visible in any logs or audits, and allow you to maintain your privilege hierarchy.
+
+      {% include integrations/templates/create-database-user-tabs.html %}
```

Comment:
The **Privileges List** tab that displays here is blank - it looks like this flavor wasn't added to the data file that creates this content. The file you want is `_data/taps/extraction/database-setup/user-privileges/mssql.yml`.

You'll also only need **one** of the two privileges that the other MSSQL databases have. One of them relates to Log-based Rep, which this integration doesn't support.